platform/lexicon.txt

```diff
@@ -0,0 +1,71 @@
+alpn
+api
+aws
+backoff
+bool
+bytestorecv
+bytestosend
+ca
+com
+const
+cwd
+dns
+endif
+errno
+filelabel
+filepath
+filepaths
+functionname
+functionpage
+functionspage
+getaddrinfo
+getcwd
+hostnamelength
+https
+ifndef
+implemenation
+inc
+iot
+logpath
+malloc
+maxattempts
+messagelevel
+mfln
```

Comment:
?

Reply:
that is a protocol, if you are confused about a word, please do a search for it in your code.